src/main/frontend/multi-pipeline-graph-view/multi-pipeline-graph/main/SingleRun.tsx

```diff
@@ -1,31 +1,51 @@
 import React, { useState } from "react";
 import { RunInfo } from "./MultiPipelineGraphModel";
 import {
+  LayoutInfo,
   PipelineGraph,
   StageInfo,
 } from "../../../pipeline-graph-view/pipeline-graph/main";
+import { defaultLayout } from "../../../pipeline-graph-view/pipeline-graph/main/PipelineGraphModel";
+import { total } from "../../../common/utils/timings";
 
 export default function SingleRun({ run, currentJobPath }: SingleRunProps) {
   const [stages, setStages] = useState<Array<StageInfo>>([]);
 
+  const layout: LayoutInfo = {
+    ...defaultLayout,
+    nodeSpacingH: 45,
+  };
+
   return (
     <tr>
       <td>
-        <a
-          href={currentJobPath + run.id}
-          className="jenkins-table__link pgw-user-specified-text"
-        >
-          {run.displayName}
-        </a>
+        <div>
+          <a
+            // style={{textOverflow: "ellipsis"}}
+            href={currentJobPath + run.id}
+            className="jenkins-table__link pgw-user-specified-text"
+          >
+            {run.displayName}
+          </a>
+        </div>
       </td>
       <td>
         <PipelineGraph
           stages={stages}
           setStages={setStages}
           currentRunPath={currentJobPath + run.id}
+          layout={layout}
           collapsed={true}
         />
       </td>
+      <td>
+        <div>
+          <span>{new Date(run.timestamp).toLocaleDateString()}</span>
+        </div>
+        <div>
+          <span className={"jenkins-subtitle"}>{total(run.duration)}</span>
```

Comment:
jenkins-subtitle doesn't exist although I think there is something, ref https://github.com/jenkinsci/design-library-plugin/issues/410